### DISCUSSION TITLE
Virtual/Local Network Gateway - da się wyłączyć rozgłaszanie poszczególnych podsieci?

### DISCUSSION THREAD

**Discussion by MariuszFerdyn** [AzureCommunityPL/group#123] [Pomoc]
A mianowicie mam kilka subnetów w sieci wirtualnej. Część z nich niestety pokrywa się z firmą, z która chce zestawić IPSEC, za pomocą Virtual Network Gateway. Niestety wszystkie te podsieci są rozgłaszane i druga strona się na to nie zgadza i zrywa połączenie (ich security).

Czy da się jakoś wyłączyć rozgłaszanie poszczególnych podsieci w Virtual Network Gateway?

Nie będzie to New-AzureRmIpsecPolicy, tylko no właśnie... jakoś słowa mi brakuje pod czym tego szukać?

**Comment by MariuszFerdyn** [comment]
We can stop propagation – we can edit the route table resources in the internal Azure subnets (or pre-do this in JSON) and disable BGP route propagation. The result of this is that the routes that the VNet Gateway were pushing out to other subnets will stop being propagated. 

Zastanawiam się, czy dla peeringów też zablokuje propagacje...

**Reply by mgrabarz** [reply]
No raczej, skoro wyłącza propagację to tym bardziej nie poleci to wgłąb peeringów.

Ja to bgp ustawiam w json na ARMach.
